pkgs/servers/web-apps/invoiceplane/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, writeText }:
+
+stdenv.mkDerivation rec {
+  pname = "invoiceplane";
+  version = "1.5.11";
+
+  src = fetchFromGitHub {
+    owner = "InvoicePlane";
+    repo = "InvoicePlane";
+    rev = "v${version}";
+    sha256 = "0cnkgp4kirsp63yipb8n1f6x3fjmyb7zjgx6sij9cpsg924sf6nx";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/invoiceplane
```

Comment:
I have also done this before... for web apps maybe simply throwing everything in `$out` is better? Maybe we should come up with an "official" policy on what to do here? I'm not sure 🤷‍♂️

Reply:
Guess that's better to put everything in $out. Even though Dokuwiki and Wordpress are doing it like this.